The tests try to read the file contents before the file is closed, so on
certain systems, those writes are still buffered, causing the file contents to
be empty.

Adding an explicit flush fixes this.
diff --git a/racksdb/tests/test_exec.py b/racksdb/tests/test_exec.py
index 08777e4..362580a 100644
--- a/racksdb/tests/test_exec.py
+++ b/racksdb/tests/test_exec.py
@@ -45,6 +45,7 @@ class TestRacksDBExec(TestRacksDBReferenceDB):
             # Check dumped DB can be loaded
             with tempfile.NamedTemporaryFile(mode="w+", suffix=".yml") as tmp:
                 tmp.write(output.getvalue())
+                tmp.flush()
                 db_dumped = RacksDB.load(schema=schema_path(), db=tmp.name)
         # Compare infrastructure in original and dumped database
         db_orig = RacksDB.load(schema=schema_path(), db=db_path())
@@ -65,6 +66,7 @@ class TestRacksDBExec(TestRacksDBReferenceDB):
             # Check DB can be loaded with schema in output
             with tempfile.NamedTemporaryFile(mode="w+", suffix=".yml") as tmp:
                 tmp.write(output.getvalue())
+                tmp.flush()
                 RacksDB.load(schema=tmp.name, db=db_path())
 
     #
diff --git a/racksdb/tests/web/test_blueprint.py b/racksdb/tests/web/test_blueprint.py
index 617f97c..3c4027d 100644
--- a/racksdb/tests/web/test_blueprint.py
+++ b/racksdb/tests/web/test_blueprint.py
@@ -97,6 +97,7 @@ class TestRacksDBWebBlueprint(TestRacksDBReferenceDB):
         # Check dumped DB can be loaded
         with tempfile.NamedTemporaryFile(mode="w+", suffix=".yml") as tmp:
             tmp.write(response.text)
+            tmp.flush()
             db_dumped = RacksDB.load(schema=self.schema_path, db=tmp.name)
         # Compare infrastructure in original and dumped database
         db_orig = RacksDB.load(schema=self.schema_path, db=self.db_path)
@@ -118,6 +119,7 @@ class TestRacksDBWebBlueprint(TestRacksDBReferenceDB):
         # Check DB can be loaded with downloaded schema
         with tempfile.NamedTemporaryFile(mode="w+", suffix=".yml") as tmp:
             tmp.write(response.text)
+            tmp.flush()
             RacksDB.load(schema=tmp.name, db=self.db_path)
 
     #
